### ISSUE TITLE
tagging scrollbar not jump to top

### ISSUE THREAD

**Issue by SoupyGit** [darktable-org/darktable#14067]
**Is your feature request related to a problem? Please describe.**
I have a long list of pre-existing tags in the tagging module. With an image selected, I like to go down the list and apply the relevant tags to the image by double clicking on the tag. The problem is that every time I add a tag by double clicking, the scrollbar jumps back to the top of list. This is annoying, as when adding tags from top to bottom there is no need to return back to the top of the list, requiring extra scrolling to re-find the previous place in the list.

**Describe the solution you'd like**
Scrollbar maintains its position (does not jump back to top) upon adding each tag.

dt 4.2.1
linux mint cinnamon 21.1

**Comment by zisoft** [comment]
I cannot reproduce that with current master.
The scrollbar position does not change when double clicking on a tag.
Tested in Lighttable as well as in Darkroom.

**Comment by SoupyGit** [comment]
For me it is the bottom list, although I see now it depends on the state of the toggle list bottom right. In one state it works correctly, but not the other.

![tagging scroll bar](https://user-images.githubusercontent.com/69561443/228994009-0bd559e1-c905-4684-88ee-21a4e9e5cf4a.png)

**Comment by zisoft** [comment]
Very strange. I am still not able to reproduce on my Mac. May this be a Linux issue?

**Comment by pehar1** [comment]
> May this be a Linux issue?

I don't think so, with 4.3.0+1485~g60a35e50e2 on Ubuntu 22.04 I can also **not** reproduce. For me the bottom list remains at the position it had at the moment of attaching the tag (double click).

before:
![tagging-before](https://user-images.githubusercontent.com/104291381/229041718-4e4e828a-afa4-4869-8340-57cae941a724.png)

after:
![tagging-after](https://user-images.githubusercontent.com/104291381/229041901-3ccfe7bd-e013-4c75-8d63-502e4ca0f262.png)

The example is with tree-view but the same holds for list-view. Possibly fixed since 4.2.1 ?

**Comment by TurboGit** [comment]
I think it reproduces if you are using a list and not the tree-view of the tag.

**Comment by pehar1** [comment]
I tried both (see last sentence of my post), also with list-view it's not reproducable for me.

**Comment by SoupyGit** [comment]
On further inspection, I think it only happens when the + button next to list/tree button is selected.

**Comment by pehar1** [comment]
This function searches the database for "suggestions" which are then displayed in the bottom list. These "suggestions" and the corresponding list are of course updated after each addition of a tag. I think therefore the jump to the top of the "suggestions" list is desired behavior in this case.
